pom.xml

```diff
@@ -68,7 +68,7 @@
     <dependencies>
         <dependency>
             <groupId>net.java.dev.jna</groupId>
-            <artifactId>jna</artifactId>
+            <artifactId>jna-jpms</artifactId>
```

Comment:
I'm mildly concerned about this change. Other codebases that leverage NuProcess may have dependency managed `jna`, but may or may not have an entry for `jna-jpms`. they'd either need to exclude it, or switch their configuration over, to avoid getting both `jna` and `jna-jpms` (and potentially at different versions).

I'm not sure there's any good way around this, here in NuProcess's POM, but at a minimum this seems like something we should at least mention in `CHANGES.md` to try and raise visibility for consumers.